assets/model_monitoring/components/src/model_data_collector_preprocessor/mdc_utils.py

```diff
@@ -155,3 +155,26 @@ def _filter_df_by_time_window(df: DataFrame, data_window_start: datetime, data_w
     """Filter dataframe on its end_time column to fit within the given data window: [start, end)."""
     df = df.filter(df.end_time >= data_window_start).filter(df.end_time < data_window_end)
     return df
+
+
+def _count_dropped_rows_with_error(
+        original_df_row_count: int,
+        transformed_df_row_count: int,
+        additional_error_msg: str = ""):
+    """Calculate number of dropped rows after transformations and throw error if too many rows lost."""
+    if original_df_row_count <= 0:
+        print("Original df contains no rows. Returning without calculating drop rate.")
+        return
+    drop_rate = abs(transformed_df_row_count - original_df_row_count) / original_df_row_count
```

Comment:
why abs? Isn't original_df_row_count always >= transformed_df_row_count?

Reply:
I think so but It was just to be safe in case someone called this function passing in bad values

Reply:
firstly, this is an internal API, we should ensure we don't make such mistake, secondly, if that really happen, abs won't ensure a correct value, we should just raise error.

Reply:
Okay that makes sense. I'll remove abs()